TMBuilder/src/main/java/com/ebay/tool/thinmodelgen/gui/menu/TMBuilderMenu.java

```diff
@@ -815,11 +827,15 @@ private void doDeveloperMockExportToStoredFile() {
   private void writeExportToFile() {
 
     ThinModelExport thinModelExport = new ThinModelExport();
+    KotlinThinModelExport kotlinThinModelExport = new KotlinThinModelExport();
     String tmbFilePath = "";
 
     try {
       tmbFilePath = currentExportFile.getCanonicalPath();
-      thinModelExport.export(currentExportFile, getValidationSetCacheAsModel());
+      if(RuntimeConfigManager.getInstance().getPlatform().equals(Platform.ANDROID))
```

Comment:
RuntimeConfigManager is specific to NST and not the TMBuilder. You will have to find an alternative option to this besides using the RuntimeConfigManager.

Reply:
I was able to achieve it by using selected file format,